CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewUtilities.java

```diff
@@ -292,8 +296,10 @@ public void execute(final PluginGraphs graphs, final PluginInteraction interacti
                     final List<ObservableList<String>> data = table.getItems();
                     writeRecords(sheet, visibleIndices, data);
                 }
-
-                workbook.write(new FileOutputStream(file));
+                FileOutputStream fileStream = new FileOutputStream(file);
+                workbook.write(fileStream);
+                fileStream.close();
```

Comment:
Thank you for fixing this serious bug.

Note that in Java 8 try with resource was introduced which will close the stream for you automatically (see https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) so you can do:

```
try (FileOutputStream fileStream = new FileOutputStream(file)) {
    workbook.write(fileStream);
}
```